orbit/pkg/packaging/deb.go

```diff
@@ -5,5 +5,5 @@ import "github.com/goreleaser/nfpm/v2/deb"
 // BuildDeb builds a .deb package
 // Note: this function is not safe for concurrent use
 func BuildDeb(opt Options) (string, error) {
-	return buildNFPM(opt, deb.Default)
+	return buildNFPM(opt, deb.Default, "deb")
```

Comment:
Needed as the replacement for `ExpandContentGlobs` (i.e. `PrepareForPackager`) expects the name of the packager to be passed in, and I didn't see a way to get it from the packager var itself.